### PR TITLE
Handle location results

### PR DESCRIPTION
**Pull request by sleepycat**
This commit adds the functionality to warn of empty results in place
while redirecting to a results page when there is something to display.

### CODE REVIEW

File: src/components/Results.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import { connect } from 'react-redux'
+import { compose, withApollo } from 'react-apollo'
+import { Trans } from 'lingui-react'
+import Breadcrumbs from './Breadcrumbs'
+import Flash from './Flash' // eslint-disable-line import/no-named-as-default
+import { Header1, Header3, PageBody } from './styles'
+import DataTable from './DataTable'
+import { setFlash, saveLocationData, deleteLocationData } from '../actions'
+
+class SearchLocation extends Component {
```
Comment (pcraig3) [inline src/components/Results.js]: Should probably change the name of this Component

File: src/components/Results.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+const mapStateToProps = state => ({
+  path: state.location.pathname,
+  data: state.data.searchLocationData,
+})
+
+export default compose(
+  withApollo,
+  reduxForm({
+    form: 'searchByLocation',
+    initialValues: { heatingType: 'any' },
```
Comment (pcraig3) [inline src/components/Results.js]: This we probably don't need anymore.

File: src/components/Results.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+const mapDispatchToProps = dispatch => {
+  return {
+    save: (data, filter) => {
+      dispatch(saveLocationData(data, filter))
+    },
+    flash: (message, priority) => {
+      dispatch(setFlash(message, priority))
+    },
```
Comment (pcraig3) [inline src/components/Results.js]: Is this needed here?  I thought it's not used except on the previous page.

File: src/components/Results.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        </PageBody>
+      </main>
+    )
+  }
+}
+
+const mapDispatchToProps = dispatch => {
+  return {
+    save: (data, filter) => {
+      dispatch(saveLocationData(data, filter))
```
Comment (pcraig3) [inline src/components/Results.js]: Is this returning the data we got from the form on the last page?

File: src/components/Results.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+const mapStateToProps = state => ({
+  path: state.location.pathname,
+  data: state.data.searchLocationData,
+})
+
+export default compose(
+  withApollo,
+  reduxForm({
+    form: 'searchByLocation',
```
Comment (pcraig3) [inline src/components/Results.js]: Do we still pass the form into the results page?

File: src/components/Results.js (hunk 3 above)
Comment (sleepycat) [reply src/components/Results.js]: Yup, removing.

File: src/components/Results.js (hunk 4 above)
Comment (sleepycat) [reply src/components/Results.js]: This isn't needed anymore. Removing.

File: src/components/Results.js (hunk 5 above)
Comment (sleepycat) [reply src/components/Results.js]: No, we are pulling from the store. Removing.

**Pull request merged**